workflows/src/main/kotlin/com/template/flows/DenyAppointmentRequest.kt

```diff
@@ -21,7 +23,7 @@ import java.util.stream.Collectors
 @InitiatingFlow
 @StartableByRPC
 class DenyAppointmentRequest(private val alice: Party,
```

Comment:
What if it's Bob who created a request? Also, I think you don't need to pass a party as a parameter here, you can get the patient from the request.